docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -299,6 +300,14 @@ testcafe chrome my-tests --app 'node server.js'
 
 Use the [--app-init-delay](#--app-init-delay-ms) option to specify the amount of time allowed for this command to initialize the tested application.
 
+### -d, --debug-mode
+
+Specify this option to run tests in the debugging mode. In this mode, test execution is paused before the first action or assertion allowing you to invoke the developer tools and debug.
```

Comment:
Yep, it works for `test.before`/ `test.after` and `fixture.beforeEach`/`fixture.afterEach`